web/src/main/kotlin/no/nav/su/se/bakover/web/routes/sak/SakRoutes.kt

```diff
@@ -24,17 +25,32 @@ internal fun Route.sakRoutes(
     sakService: SakService
 ) {
     get(sakPath) {
-        call.lesFnr("fnr").fold(
-            ifLeft = { call.svar(BadRequest.message(it)) },
-            ifRight = { fnr ->
-                sakService.hentSak(fnr)
-                    .mapLeft { call.svar(NotFound.message("Fant ikke noen sak for person: $fnr")) }
-                    .map {
-                        call.audit("Hentet sak for fnr: $fnr")
-                        call.svar(Resultat.json(OK, serialize((it.toJson()))))
+        when {
+            call.parameters.contains("saksnummer") -> {
+                call.withSaksnummer { saksnummer ->
+                    call.svar(
+                        sakService.hentSak(saksnummer).fold(
+                            { NotFound.message("Fant ikke sak med saksnummer: $saksnummer") },
+                            { Resultat.json(OK, serialize((it.toJson()))) }
+                        )
+                    )
+                }
+            }
+            call.parameters.contains("fnr") -> {
+                call.lesFnr("fnr").fold(
+                    ifLeft = { call.svar(BadRequest.message(it)) },
+                    ifRight = { fnr ->
+                        sakService.hentSak(fnr)
+                            .mapLeft { call.svar(NotFound.message("Fant ikke noen sak for person: $fnr")) }
```

Comment:
Hva liker du best her? fold eller mapLeft og map? Ref
>sakService.hentSak(saksnummer).fold(
                            { NotFound.message("Fant ikke sak med saksnummer: $saksnummer") },
                            { Resultat.json(OK, serialize((it.toJson()))) }
                        )